content/03.results.md

```diff
@@ -299,7 +299,7 @@ We further reproduced previous findings (**Figure {@fig:Fig5}E**) that medullobl
 However, we also found that higher expression of _CD274_ was significantly associated with improved overall prognosis for medulloblastoma tumors, although with a marginal effect size (HR = 0.0012, 95% CI = 7.5e−06 - 0.18, p = 0.008, multivariate Cox) (**Figure {@fig:Fig5}D**).
 This result may be explained by the higher expression of _CD274_ found in WNT subtype tumors by us and others [@doi:10.1080/2162402X.2018.1462430], as this diagnosis carries the best prognosis of all medulloblastoma subgroups (**Figure {@fig:Fig5}E**).
 
-Finally, we asked whether any molecular subtypes might show an immunologically-hot phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells [@doi:10.1136/jitc-2021-004012; @doi:10.4236/jct.2013.48164].
-While adamantinomatous craniopharyngiomas and Group 3 and Group 4 medulloblastomas had the highest CD8+ to CD4+ T cell ratios (**Figure {@fig:S6}F**), very few tumors had ratios greater than 1, highlighting an urgent need to identify novel therapeutics for these immunologically-cold pediatric brain tumors with poor prognosis.
+Finally, we asked whether any molecular subtypes might have a high ratio CD8+ to CD4+ T cells, a metric which has been associated with better immunotherapy response and prognosis following PD-L1 inhibition in non-small cell lung cancer or adoptive T cell therapy in multiple stage III or IV cancers [@doi:10.1136/jitc-2021-004012; @doi:10.4236/jct.2013.48164].
```

Comment:
If we want to keep hot/cold phrasing in there somewhere, I think it would be fine to add in something along those lines "Tumors with this high ratio are considered to have an 'immunologically-hot' phenotype, in contrast to 'immunologically-cold' tumors."

Reply:
I think it is ok to remove and be more specific about what it means (ie not going to respond to xy therapy) potentially